src/test/java/com/wikia/webdriver/testcases/desktop/userprofile/UserAvatar.java

```diff
@@ -83,4 +83,22 @@ public void staffUserCanRemoveAvatar() {
     Assertion.assertNotEquals(changedAvatarUrl, avatarUrl);
     changedProfile.verifyURLStatus(200, changedAvatarUrl);
   }
+
+  @Test(groups = "UserAvatar_staffUserCanSelectDefaultAvatar")
+  @Execute(asUser = User.SUS_STAFF)
+  public void staffUserCanSelectDefaultAvatar() {
+    UserProfilePage profile = new UserProfilePage().open(User.SUS_STAFF.getUserName());
+    AvatarComponentObject avatar = profile.clickEditAvatar();
+    profile.verifyAvatar();
+
+    String avatarUrl = profile.getAvatarImageSrc();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "avatarUrl" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)